front/lib/api/files/upload.ts

```diff
@@ -243,8 +297,15 @@ const extractContentAndSchemaFromCSV: PreprocessingFunction = async (
     });
 
     // Process the first stream for processed file
+    let content: string = "";
     const processedPipeline = pipeline(
       readStream.pipe(new PassThrough()),
+      async function* (source) {
```

Comment:
The goal of this API was to never load the entire file content in memory. Even though some pre-processing action will do it, this part was meant to handle bigger files. Not sure to understand why we need the content here.